### PR TITLE
Issue #116 - do not generate schema when @RequestBody is a reference

### PR DESCRIPTION
**Pull request by MikeEdgar**
Prevent auto-generation of a `Schema` when a `@RequestBody` annotation contains a reference.

### CODE REVIEW

**Comment by MikeEdgar** [comment]
Strange build failure in semaphore. The TCK tests passed.

**Comment by kenfinnigan** [comment]
Forked JVM crashed. Trying a rebuild

File: implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java [hunk 1]
```diff
@@ -609,7 +609,9 @@ private void processJaxRsMethod(OpenAPIImpl openApi, ClassInfo resource, MethodI
         for (AnnotationInstance annotation : requestBodyAnnotations) {
             RequestBody requestBody = readRequestBody(annotation);
             // TODO if the method argument type is Request, don't generate a Schema!
-            if (!ModelUtil.requestBodyHasSchema(requestBody)) {
+
+            // Only generate the request body schema is the @RequestBody is not a reference and no schema is yet specified
```
Comment (EricWittmann) [inline implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java]: BTW this is the typo that was mentioned in the issue thread.

**Comment by MikeEdgar** [comment]
The comment typo has been fixed and an additional test scenario added.

**Pull request merged**